.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+app/node_modules/
+script/changelog/node_modules
```

Comment:
I’m pretty sure `node_modules` directories are ignored by default.

Reply:
This got caught by

https://github.com/desktop/desktop/blob/d73112a9fc9ca4a953697ae471d3942ae8e07d7b/script/eslint.ts#L17